examples/fc_gauss_analytical.py

```diff
@@ -0,0 +1,49 @@
+"""Analytical solution for Gaussian with a boundary at
+   the origin. Produces Fig. 10 from the Feldman Cousins
+   paper."""
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy import stats
+
+from gammapy.stats import (
+    fc_find_acceptance_region_gauss,
+    fc_fix_upper_and_lower_limit,
+)
+
+fSigma       = 1
+fStepWidthMu = 0.005
+fMuMin       = 0
+fMuMax       = 8
+fNSigma      = 10
+fNStep       = 1000
+fCL          = 0.90
+
+XBins  = np.linspace(-fNSigma*fSigma, fNSigma*fSigma, fNStep, endpoint=True)
+MuBins = np.linspace(fMuMin, fMuMax, fMuMax/fStepWidthMu + 1, endpoint=True)
+
+matrix = [dist/sum(dist) for dist in (stats.norm(loc=mu, scale=fSigma).pdf(XBins) for mu in MuBins)]
+
+UpperLimitAna = []
+LowerLimitAna = []
+
+for mu in MuBins:
```

Comment:
This example runs for several minutes without feedback to the user.
They might think it's broken (infinite loop) or something, or get annoyed at the long wait and, god forbid, try to `import ROOT`.

How about showing a progressbar?

Put this at the top:
```python
from astropy.utils.console import ProgressBar
```
and then just changing this line to:
```
for mu in ProgressBar(MuBins):
```

Also, even if it doesn't look like the plot in the FC paper anymore, I'd suggest you change the example (e.g. fewer bins) so that it runs in seconds instead of minutes.